libraries/Ethernet/src/ETH.cpp

```diff
@@ -861,24 +861,42 @@ bool ETHClass::setHostname(const char * hostname)
     return esp_netif_set_hostname(_esp_netif, hostname) == 0;
 }
 
-bool ETHClass::enableIpV6()
+bool ETHClass::enableIPv6(bool en)
 {
-    if(_esp_netif == NULL){
-        return false;
+    // if(_esp_netif == NULL){
+    //     return false;
+    // }
+    // return esp_netif_create_ip6_linklocal(_esp_netif) == 0;
```

Comment:
This should be removed or add a comment explaining why it is there

Reply:
this code will be refactored as soon as it's merged. It's there for remindder